core/test/base/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set (link_libs Core)
+
+ROOT_ADD_GTEST(CoreBaseTests
```

Comment:
I agree. It makes much more sense to have the test for a module under that module's directory rather than under the category (and core is a bad example to talk about the differences between the two :)).